msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -414,24 +280,26 @@ class NativeAuthPublicClientApplication(
      * Sign in the account using username and password; Kotlin coroutines variant.
      *
      * @param username username of the account to sign in.
-     * @param password password of the account to sign in.
+     * @param password (Optional) password of the account to sign in.
```

Comment:
This needs to be removed, as the optionality doesn't apply to Java call sites. Applies in different places in the code.

Reply:
The place where the optionality doesnot apply is in an internal package and not used from the Sample app. Optional in the doc should stay here as it provides meaningful information to the developer.